app/services/gogox.rb

```diff
@@ -0,0 +1,129 @@
+class Gogox
+
+  attr_accessor :params, :time, :vehicle, :district_id
+
+  VEHICLE_TYPES = ["van", "mudou", "mudou9"]
+
+  def initialize(options = {})
+    @params        = options
+    @time         = parse_pickup_time(options[:scheduled_at])
+    @vehicle      = options[:vehicle_type]
+    @district_id  = options[:district_id]
```

Comment:
Is the `@district_id` instance variable used in this class somehow?

Reply:
yes, not used, will remove it.